driver/level3/trmm_L.c

```diff
@@ -397,9 +426,13 @@ int CNAME(blas_arg_t *args, BLASLONG *range_m, BLASLONG *range_n, FLOAT *sa, FLO
 	STOP_RPCC(trmmcost);
       }
 
-      for(is = ls - min_l + min_i; is < ls; is += GEMM_P){
+      for(is = ls - min_l + min_i; is < ls; is += min_i){
 	min_i = ls - is;
 	if (min_i > GEMM_P) min_i = GEMM_P;
+	if (min_i > GEMM_UNROLL_M){
+	    min_i = (min_i / GEMM_UNROLL_M) * GEMM_UNROLL_M;
```

Comment:
Do you mean `min_i=min_i % GEMM_UNROLL_M` ?

Reply:
The two are obviously not equivalent. min_i = (min_i / GEMM_UNROLL_M) * GEMM_UNROLL_M means rounding down to UNROLL_M.

Reply:
you can test strmm_kernel_16x4.S to verify my merge.  Set SGEMM_DEFAULT_P  in param.h to 28 and set the scale of the test matrix to m = n = k = 44 in ctest/sin3 . Then, you'll find that ctest fails as follows:
![imng](https://github.com/xiaojiayuan111/resources/blob/master/error.PNG?raw=true)
more details as follows:
https://groups.google.com/forum/#!topic/openblas-dev/O2NemfVkrI8